rskj-core/src/main/java/co/rsk/core/bc/SelectionRule.java

```diff
@@ -0,0 +1,54 @@
+package co.rsk.core.bc;
+
+import co.rsk.remasc.Sibling;
+import org.ethereum.core.Block;
+import org.ethereum.core.BlockHeader;
+import org.ethereum.util.FastByteComparisons;
+
+import java.math.BigInteger;
+import java.util.List;
+
+public class SelectionRule {
+
+    private static final int BYTE_ARRAY_OFFSET = 0;
+    private static final int BYTE_ARRAY_LENGTH = 32;
+
+    private static final int PAID_FEES_MULTIPLIER_CRITERIA = 2;
+
+    public static boolean shouldWeAddThisBlock(BigInteger blockDifficulty, BigInteger currentDifficulty,
+        Block block, Block currentBlock) {
+        int compareDifficulties = blockDifficulty.compareTo(currentDifficulty);
+
+        if (compareDifficulties > 0) {
+            return true;
+        }
+
+        if (compareDifficulties == 0) {
+            if (block.getHeader().getPaidFees() >
```

Comment:
Please make two different ifs for readability

Reply:
Done